app/main.py

```diff
@@ -1,3 +1,18 @@
 class OnlineCourse:
-    # write your code here
-    pass
+    @classmethod
+    def from_dict(cls, course_dict):
+        return OnlineCourse(name=course_dict.get("name"),
+                            description=course_dict.get("description"),
+                            weeks=cls.days_to_weeks(course_dict.get("days")))
+
+    @staticmethod
+    def days_to_weeks(days):
+        return days // 7 if days % 7 == 0 else days // 7 + 1
+
+    def __init__(self, *args, **kwargs):
```

Comment:
Why init method has such a structure?
`OnlineCourse` constructor should take three arguments